CampusTour/ViewControllers/FeaturedViewController.swift

```diff
@@ -214,37 +241,52 @@ class FeaturedViewController: UIViewController, FilterFunctionsDelegate, PopupFi
     func setItemFeedDefaultSpec() {
         itemFeedViewController.updateItems(newSpec: ItemFeedSpec.getSharedDataSpec())
     }
-
 }
 
 extension FeaturedViewController: ItemFeedSearchManagerDelegate {
     func didStartSearchMode() {
+        //remove cancel button -- doesn't work in ItemFeedSearchManager
+        if let sb = navigationItem.titleView as? UISearchBar { sb.showsCancelButton = false }
```

Comment:
Does `sb.showsCancelButton = false` work? There's also a function `sb. setShowsCancelButton(false, animated: true)`

Reply:
Removing cancel button works, but only when I put the code in FeaturedVC. Oddly modifying searchbar in ItemFeedSearchManager didn't reflect the changes correctly